icons/soup.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <path d="M12 21c6.5 0 9-4.03 9-9H3c0 4.97 2.5 9 9 9Z"/>
+ <path d="M20 11.5 22 6"/>
+ <path d="M11.21 2.972c.326.129.958.637.888 1.645-.07 1.008-1.104 1.455-1.174 2.45-.066.946.396 1.506.866 1.96"/>
+ <path d="M6.21 2.972c.326.129.958.637.888 1.645-.07 1.008-1.104 1.455-1.174 2.45-.066.946.396 1.506.866 1.96"/>
+ <path d="M16.21 2.972c.326.129.958.637.888 1.645-.07 1.008-1.104 1.455-1.174 2.45-.066.946.396 1.506.866 1.96"/>
+ <path d="M7 21h10"/></svg>
```

Comment:
Love the design, although once again, I think a more elliptical bowl shape would fit the Lucide style better, and there are some guideline violations and misaligned paths, my suggestion would be:
![image](https://user-images.githubusercontent.com/17746067/199519465-bb3fea21-1545-4d9f-9050-a00f6f674477.png)
![soup](https://user-images.githubusercontent.com/17746067/199519358-6ada2845-d494-41c2-ba31-431aad3687ac.png)

```suggestion
  <path d="M12 21a9 9 0 0 0 9-9H3a9 9 0 0 0 9 9Z" />
  <path d="M19.5 12 22 6" />
  <path d="M11.25 3c.27.1.8.53.75 1.36-.06.83-.94 1.2-1 2.02-.05.78.34 1.25.74 1.62" />
  <path d="M6.25 3c.27.1.8.53.75 1.36-.06.83-.94 1.2-1 2.02-.05.78.34 1.25.74 1.62" />
  <path d="M16.25 3c.27.1.8.53.75 1.36-.06.83-.94 1.2-1 2.02-.05.78.34 1.25.74 1.62" />
  <path d="M7 21h10" />
</svg>
```